lib/widgets/featured_stories_screen/stories/carousel/card/card_story_preview.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_layout/assets/constants.dart';
+
+class CardStoryPreview extends StatelessWidget {
+  const CardStoryPreview({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    Widget storyPreviewTitle = Container(
+        padding: const EdgeInsets.only(bottom: 1),
+        child: const Text(
+          FAKE_STORY_TITLE,
+          style: TextStyle(fontWeight: FontWeight.w500, fontSize: 17),
+        ));
+
+    Widget storyPreviewDescription = Text(FAKE_STORY_DESCRIPTION,
+        style: TextStyle(color: Colors.black.withOpacity(0.6)));
+
+    return Container(
+      padding: const EdgeInsets.only(top: 15, left: 15),
```

Comment:
I am not sure that it works the way I expect it to, but there is a parameter in EdgeInsets called topLeft, and it looks like it would be more semantic. So it will help you in that case.
Please, correct me if I am wrong, it is only my assumption.

Reply:
Sounds great; I've tried to use that parameter but cannot come up with the correct implementation for it. I've tried out various options, read the documentation (unfortunately, there are no examples), and can't determine which constructor of `EdgeInsets` could be suitable for the `topLeft `parameter. Please, provide me with some examples.